### DISCUSSION TITLE
Adding a menu in the lower bottom corner of Visual Studio 2022 IDE

### DISCUSSION THREAD

**Discussion by yonidavidson** [VsixCommunity/Community.VisualStudio.Toolkit#449] [Q&A] [answered]
I would like to add a menu for my extension here: 
![image](https://github.com/VsixCommunity/Community.VisualStudio.Toolkit/assets/5472778/9d10a1f6-6b83-41a8-a36d-35914698b3f4)

can someone help me? managed to create toolbars everywhere but there.

**Comment by ErikEJ** [comment]
You can do this (limited): https://learn.microsoft.com/en-us/visualstudio/extensibility/extending-the-status-bar?view=vs-2022

**Reply by yonidavidson** [reply]
Can I use that to get a menu next to the intelicode/ error menu ?

**Reply by ErikEJ** [reply]
No

**Reply by yonidavidson** [reply]
Thanks, but I want to add a menu there, for now I am not even sure how its called

**Reply by ErikEJ** [reply]
It is not possible

**Reply by yonidavidson** [reply]
So how do I see there all the other menus? Intelicode, Github-Copilot, Cleanup/Errors etc?

**Reply by mrlacey** [reply, marked as answer]
> So how do I see there all the other menus? Intelicode, Github-Copilot, Cleanup/Errors etc?

As those are Microsoft-owned tools/extensions and there aren't any official/public/supported extension points for this functionality, I assume they're using private and/or undocumented APIs to do this. There are lots of other things that VS (& some MS-provided tools) do that aren't made available to 3rd party extension developers via public APIs.
